README.md

```diff
@@ -6,3 +6,21 @@ Pulp 3 Ansible Installer
 A collection of roles to install or upgrade Pulp 3. Can also reconfigure or add plugins to an existing install.
 
 See https://pulp-installer.readthedocs.io/ for full documentation.
+
+
+How to File an Issue
+--------------------
+
+[New installer issue](https://pulp.plan.io/projects/pulp/issues/new)
```

Comment:
Can this be rewritten to `To file a new issue set the Category to 'Installer' when filing [here](https://pulp.plan.io/projects/pulp/issues/new).`.